pkg/oci/load_balancer_security_lists_test.go

```diff
@@ -21,18 +21,20 @@ import (
 	"github.com/oracle/oci-go-sdk/common"
 	"github.com/oracle/oci-go-sdk/core"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Minor: This is generally inconsistent in this PR. Sometimes it's qualified sometimes not. We don't need to qualify so suggest removing.